### PR TITLE
Support .env file

### PR DESCRIPTION
**Pull request by dyadyaJora**
Let's use `.env` file for saving and controlling environment variables.

There is `.env.sample` file where we have to describe all env vars.
Before running project - create `.env` file from the sample with command `cp .env.sample .env` and fill `.env` file with our private credentials.
In docker this file will be used automatically
On develop environment you may run `source .env` to import all env var from `.env` file

### CODE REVIEW

File: Dockerfile-frankyapp [hunk 1]
```diff
@@ -21,6 +21,10 @@ ENV DOCKER_NETWORK_HOST=0.0.0.0
 
 COPY --from=builder /install /usr/local
 COPY ./backend /home/runner/franky/backend
+
+
+# COPY ./backend/private-key.pem /home/runner/franky/backend/private-key.pem
```
Comment (tcibinan) [inline Dockerfile-frankyapp]: Probably, this is some debug staff which can be excluded from the final docker file.

File: docker-compose.yml [hunk 2]
```diff
@@ -6,6 +6,8 @@ services:
     build:
       context:  ./
       dockerfile: Dockerfile-frankyapp
+    env_file:
+      - .env
```
Comment (tcibinan) [inline docker-compose.yml:12]: Fix me if I wrong but these configuration can be omitted. `.env` is a default environment file.

File: docker-stack.yml [hunk 3]
```diff
@@ -3,6 +3,8 @@ version: '3'
 services:
   frankyapp:
     image: franky_frankyapp:latest
+    env_file:
+      - .env
```
Comment (tcibinan) [inline docker-stack.yml]: I didn't noticed it before but why is there a need for both `docker-compose.yml` and `docker-stack.yml` files? I know nothing about the docker stack technology but according to the [docs](https://docs.docker.com/get-started/part5/) `docker-compose.yml` can be used with stack operations at least at some cases.

Probably, we can merge  `docker-compose.yml` and `docker-stack.yml` into a single file. What do you think?

**Review by tcibinan — CHANGES_REQUESTED** [review]
Thank for the pull request.

Could you please answer a few questions below.

File: docker-compose.yml (hunk 2 above)
Comment (dyadyaJora) [reply docker-compose.yml:12]: dont work without this lines

File: docker-stack.yml (hunk 3 above)
Comment (dyadyaJora) [reply docker-stack.yml]: +++

File: Dockerfile-frankyapp (hunk 1 above)
Comment (dyadyaJora) [reply Dockerfile-frankyapp]: +++

File: docker-compose.yml (hunk 2 above)
Comment (tcibinan) [reply docker-compose.yml:12]: Ok, lets leave it as is.

**Review by tcibinan — APPROVED** [review]

**Review by lerushe — APPROVED** [review]

**Pull request merged**